Framework/DataHandling/inc/MantidDataHandling/LoadEventNexus.h

```diff
@@ -17,8 +17,11 @@
 #undef CHAR
 #endif
 
-#include <nexus/NeXusException.hpp>
+// clang-format off
 #include <nexus/NeXusFile.hpp>
+#include <nexus/NeXusException.hpp>
+// clang-format on
+#include <nexus/NeXusException.hpp>
```

Comment:
Why include this twice?

Reply:
Nice catch. I did this with a script and didn't check all of the files by hand.